src/room/InCallView.tsx

```diff
@@ -369,52 +333,74 @@ export const InCallView: FC<InCallViewProps> = ({
           />
         );
       }),
-    [
-      toggleFullscreen,
-      toggleSpotlightFullscreen,
-      openProfile,
-      showSpeakingIndicators,
-      showSpotlightIndicators,
-    ],
+    [vm, toggleSpotlightExpanded, openProfile],
   );
 
+  const layouts = useMemo(() => {
+    const inputs = {
+      minBounds: gridBoundsObservable,
+      spotlightAlignment,
+      pipAlignment,
+    };
+    return {
+      grid: makeGridLayout(inputs),
+      "spotlight landscape": makeSpotlightLandscapeLayout(inputs),
+      "spotlight portrait": makeSpotlightPortraitLayout(inputs),
+      "spotlight expanded": makeSpotlightExpandedLayout(inputs),
+      "one-on-one": makeOneOnOneLayout(inputs),
```

Comment:
I think it would be nice to use a consistent style for all of those layouts:
```
      "spotlight-landscape": makeSpotlightLandscapeLayout(inputs),
      "spotlight-portrait": makeSpotlightPortraitLayout(inputs),
      "spotlight-expanded": makeSpotlightExpandedLayout(inputs),
      "one-on-one": makeOneOnOneLayout(inputs),
```